src/RuleSet/DeclarationBlock.php

```diff
@@ -45,18 +45,18 @@ public static function parse(ParserState $parserState, $list = null)
         $result = new DeclarationBlock($parserState->currentLine());
         try {
             $aSelectorParts = [];
-            $sStringWrapperChar = false;
+            $stringWrapperChar = false;
```

Comment:
As we're renaming this variable anyway, could we not change it to `$stringWrapperCharacter` in this PR?

I note a couple of other very minor issues with this section of code that are beyond the scope of this PR, but should be tidied up (and can't be until this PR is merged):
1. The non-set value for the above variable should be `null` rather then `false`;
2. The comparison in the `elseif` should be strict (using `===`).

Reply:
I have create #975 and #976 for those two issues.